egs/wsj/asr2/run.sh

```diff
@@ -0,0 +1,370 @@
+#!/bin/bash
+
+# Copyright 2017 Johns Hopkins University (Shinji Watanabe)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh
+. ./cmd.sh
+
+# general configuration
+backend=chainer
+stage=0        # start from 0 if you need to start from data preparation
+stop_stage=100
+ngpu=0         # number of gpus ("0" uses cpu, otherwise use gpu)
+debugmode=1
+dumpdir=dump   # directory to dump full features
+N=0            # number of minibatches to be used (mainly for debugging). "0" uses all minibatches.
+verbose=0      # verbose option
+resume=        # Resume the training from snapshot
+seed=1
+
+# feature configuration
+do_delta=false
+
+# network architecture
+ninit=pytorch
+ntype=transformer
```

Comment:
I think $ntype is only used in $expdir and it does not change `asr_train.py` behavior. How about new cmd arg `mode_lmodule="espnet.nets.${backend}_backend.asr_transformer.E2E"` instead to indicate the exact model?